adoptedStyleSheets.js

```diff
@@ -285,7 +323,7 @@
         throw new TypeError('Adopted style sheets must be an Array');
       }
 
-      if (!sheets.every(sheet => sheet instanceof OldCSSStyleSheet)) {
+      if (!sheets.every(sheet => sheet instanceof OldCSSStyleSheet || sheet.constructor === ConstructStyleSheet)) {
```

Comment:
Hmm, not sure I follow the idea of this change. As I understand, we can create a sheet that has `ConstructStyleSheet` as a constructor only via the `Reflect.construct()` which is not the very common case. Am I missing something?